### PR TITLE
KEP-1525 Create debian installer

### PR DESCRIPTION
**Pull request by ebruck**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (nickroeker) [inline LICENSE]: I believe this is to be Apache 2.0, not AGPL, since it's a client shim and the clients are Apache 2.0. @amastracci?

Comment (amastracci) [reply LICENSE]: Confirmed. All client libs should be licensed under apache2.0

File: CMakeLists.txt [hunk 2]
```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.10)
 
 project(bzapi CXX)
 
+# for now...
```
Comment (amastracci) [inline CMakeLists.txt]: I assume this means we will move to a separate CMakeLists.txt in the future as we do with SwarmDB

**Review by amastracci — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (ebruck) [reply CMakeLists.txt]: @amastracci Then the source code will also need the licence cleaned up. @paularchard please create a task for this.

File: LICENSE (hunk 1 above)
Comment (ebruck) [reply LICENSE]: @amastracci Then the source code will also need the licence cleaned up. @paularchard please create a task for this.

File: CMakeLists.txt (hunk 2 above)
Comment (ebruck) [reply CMakeLists.txt]: yes... eventually.

**Pull request merged**